src/components/syncPlay/ui/groupSelectionMenu.css

```diff
@@ -0,0 +1,4 @@
+.syncPlayGroupMenu .actionSheetText {
```

Comment:
I don't think this was the case when you made this change, but we have moved to using sass everywhere (except for themes) so this should have a `.scss` extension.